CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ Multitenant AWS Dynamo supports the [AWS Dynamo Java API](http://docs.aws.amazon
   
 You can write your application code against the Amazon DynamoDB interface as you would for any other application.  The implementation will manage storage of data by tenant.
 
+## 0.9.26 (December 6, 2018)
+
+* Fixed bug in `SharedTable` where `.` in a table name would cause hash keys to be misread
```

Comment:
`'.'`
instead of
`.`
?

Reply:
fixed in 0fb54fe8ce22d3528bb416f873c4ed65f631fb48